features/step_definitions.rb

```diff
@@ -92,27 +92,27 @@
 
 #
 
-Given %r{^I have a configuration file with "(.*)" set to "(.*)"$} do |key, value|
+Given(/^I have a configuration file with "(.*)" set to "(.*)"$/) do |key, value|
   config = if source_dir.join("_config.yml").exist?
-    SafeYAML.load_file(source_dir.join("_config.yml"))
-  else
-    {}
-  end
+             SafeYAML.load_file(source_dir.join("_config.yml"))
```

Comment:
That's over tabbed and breaks readability, no matter what Rubocop tries to tell you and because it's not customizable more in this aspect we do:

```ruby
val = \
  if blah
    work(
      here
    )
  end
```